transportable-udfs-type-system/src/main/java/com/linkedin/transport/typesystem/TypeSignature.java

```diff
@@ -159,6 +159,15 @@ private static TypeSignatureElement getTypeSignatureElement(String currentBase)
         case "varchar":
           currentBaseElement = ConcreteTypeSignatureElement.STRING;
           break;
+        case "real":
+          currentBaseElement = ConcreteTypeSignatureElement.FLOAT;
+          break;
+        case "double":
+          currentBaseElement = ConcreteTypeSignatureElement.DOUBLE;
+          break;
+        case "binary":
+          currentBaseElement = ConcreteTypeSignatureElement.BINARY;
+          break;
```

Comment:
Could you please update the repo documentation by adding description of new types to relevant sections?